addons/context_actions/stringtable.xml

```diff
@@ -16,5 +16,21 @@
             <German>Zeus teleportieren</German>
             <Polish>Teleportuj Zeusa</Polish>
         </Key>
+        <Key ID="STR_ZEN_Context_Actions_VehicleLogistics">
+            <English>Vehicle Logistics</English>
+            <Polish>Logistyka pojazdów</Polish>
+        </Key>
+        <Key ID="STR_ZEN_Context_Actions_Rearm">
+            <English>Rearm</English>
+            <Polish>Przezbrój</Polish>
+        </Key>
+        <Key ID="STR_ZEN_Context_Actions_Refuel">
+            <English>Refuel</English>
+            <Polish>Zatankuj</Polish>
+        </Key>
+        <Key ID="STR_ZEN_Context_Actions_Repair">
```

Comment:
`str_a3_showcase_jets_bis_tskrepair_wp` could be used (Uppercase)

Reply:
These all are uppercase so not really useful. Although correct strings should be somewhere in vanilla, but I don't know where to look for them.

Reply:
https://synixebrett.github.io/stringtables/